lib/quarantine/databases/dynamo_db.rb

```diff
@@ -23,23 +23,10 @@ def scan(table_name)
         result&.items
       end
 
-      def batch_write_item(table_name, items, additional_attributes = {}, dedup_keys = %w[id full_description])
-        return if items.empty?
-
-        scanned_items = scan(table_name)
-
-        deduped_items = items.reject do |item|
```

Comment:
why delete this dedup logic ? did it go to somewhere else?

Reply:
IIRC I inspected the code and determined a duplicate item shouldn't be possible. In any case, keeping this logic would have made the changes in PR more complicated.